src/types/common.ts

```diff
@@ -13,3 +13,5 @@ export interface BallChaser {
   team: Readonly<Team | null>;
   queueTime: Readonly<DateTime | null>;
 }
+
+export const normIconURL = "https://raw.githubusercontent.com/mattwells19/UNCC-Six-Mans.js/main/media/norm_still.png";
```

Comment:
I don't know if this is the best place for this tbh :thinking:. What about just putting this string inside of `LeaderboardChannelController.ts`?

The reason I say this is because this is the `types` folder and this isn't really a "type." Just a constant string. Thoughts?

Reply:
Yeah I was on the fence about that.  I went ahead and moved it into the controller.  This all might change when we refactor the embed message, but this will at least show the thumbnail piece we can leverage later.